python/py-XlsxWriter/Portfile

```diff
@@ -4,11 +4,11 @@ PortSystem          1.0
 PortGroup           python 1.0
 PortGroup           github 1.0
 
-github.setup        jmcnamara XlsxWriter 1.2.5 RELEASE_
+github.setup        jmcnamara XlsxWriter 1.2.8 RELEASE_
 name                py-XlsxWriter
 
-python.versions     27 35 36 37 38
-python.default_version 27
+python.versions     27 36 37 38
+python.default_version 38
```

Comment:
I think that the `python.default_version` line should be removed so the port defers to the default in the python portgroup.